Source/WebGPU/WebGPU/RenderPassEncoder.mm

```diff
@@ -247,9 +247,6 @@
 
 void RenderPassEncoder::setBindGroup(uint32_t groupIndex, const BindGroup& group, uint32_t dynamicOffsetCount, const uint32_t* dynamicOffsets)
 {
-    UNUSED_PARAM(dynamicOffsetCount);
```

Comment:
Not strictly related, except I found we ignored `firstIndex` by searching for `UNUSED_PARAM`